modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -40,6 +41,14 @@
 	public int doStartTag() throws JspException {
 		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
 
+		if (getPaginationBarDeltas() == null) {
+			setPaginationBarDeltas(
+				Arrays.asList(
+					new PaginationBarDelta(10), new PaginationBarDelta(20),
+					new PaginationBarDelta(30), new PaginationBarDelta(50)));
+			_showDropDown = false;
+		}
```

Comment:
I'm not sure I understand the desired behavior right, but if we don't want to show DropDown of deltas, then we shouldn't pass the `deltas`, this sounds a bit contradictory. As I commented on https://github.com/liferay/clay/pull/4178 I think we can simplify and maintain behavior compatibility.

Reply:
I also think we can simplify.

> if we don't want to show DropDown of deltas, then we shouldn't pass the `deltas`, this sounds a bit contradictory.

The thing is, on the React side, if you don't pass `deltas`, it still get assigned a [default value](https://github.com/liferay/clay/blob/d8251c47262ccff9e715baf050c2713b075b11fa/packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx#L119), which is then used for `activeDeltas` [if not passed](https://github.com/liferay/clay/blob/d8251c47262ccff9e715baf050c2713b075b11fa/packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx#L130-L132) (i.e. it's already "confusing" enough) 

I didn't see what we're breaking here, but I think I get your point: now if you don't pass `deltas` (or `paginationBarDeltas` on the tag side) we're still going to assign the same default value **but** additionally we're not going to show the dropdown: and that's what you probably mean by "breaking behavior"

In terms of simplifying how about forgetting this change and adding a new (boolean) prop/attribute to avoid rendering the dropdown?

Reply:
Yeah, I think changing this behavior would change a few things if we was going to make it the default on the React side, we would have to stop relying on `deltas` and remove usage of the default value and change some `maps` that it uses internally. In this case, I would just go with the prop, I think it would simplify the whole implementation for now.